### PR TITLE
Add C package xdotool recipe

### PR DESCRIPTION
**Pull request by ehfd**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ehfd** [comment]
@conda-forge/help-c-cpp Ready for review.

Note:
- Linux-only because libxkbcommon and xkeyboard-config are Linux-only, They should be able to be built for Mac and Windows, but I have a short timeframe so it would be tedious for me to PR every feedstock when I still have a lot of packages left.
- xorg-libxtst feedstock should have a run_export to it, but this should be done by pkgw at once instead of PRing every time. I will remove the runtime dependency when that happens.

**Comment by ehfd** [comment]
@hmaarrfk Since you're the maintainer of libxkbcommon, is there a specific reason this is only on Linux?

**Comment by hmaarrfk** [comment]
No reason other than nobody necessarily needed it (until maybe now). Generally if it isn't tested it is broken. So we are often cautious with these OSX X11 packages since sometimes they can have negative effects for the end users since they might change Wich graphics frontend is chosen.

**Comment by hmaarrfk** [comment]
PRs welcome

**Comment by ehfd** [comment]
> PRs welcome

Eventually. @hmaarrfk 

For now, I'll keep this Linux-only then.

**Comment by hmaarrfk** [comment]
Whatever is easiest for your desired outcomes!!!

**Comment by ehfd** [comment]
Anyways, thanks for your dedication! @hmaarrfk @h-vetinari

File: recipes/xdotool/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - make
+    - pkg-config
+  host:
+    - xorg-libx11
+    - xorg-libxi
+    - xorg-libxtst
+    - xorg-libxinerama
+    - libxkbcommon
+  run:
+    - xorg-libxtst
```
Comment (hmaarrfk) [inline recipes/xdotool/meta.yaml:29]: Should we add a run export here?

**Comment by hmaarrfk** [comment]
run export can be added later thx

File: recipes/xdotool/meta.yaml (hunk 1 above)
Comment (ehfd) [reply recipes/xdotool/meta.yaml:29]: I already have the PR. @hmaarrfk 
https://github.com/conda-forge/xorg-libxtst-feedstock/pull/9

**Pull request merged**